### ISSUE TITLE
https streams not working after server move

### ISSUE THREAD

**Issue by bombelman** [AzuraCast/AzuraCast#7501]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

 v0.20.2 Stable • Docker • PHP 8.3

### What happened?

Hi all,

I moved my AzuraCast install to a new server (will post about that later),
however the https streams are not working.

While the https domain certificate is working fine (e.g. https://test.example.com),
the https is not working for the stream domains with port (e.g. https://test.example.com:8000)

the domains are already in system settings and have working certs, just not for the streaming ports.
How to fix?
Any idea? Thanks

### Relevant log output

```shell
which log?
```

**Comment by bombelman** [comment]
I REALLY need to get this fixed, my stream server is useless now really.
My previous posts on this forum went unanswered or unresolved.

If anyone has an idea, feel free to also contact me in private, willing to pay to have this fixed if there is no other way.

**Comment by bombelman** [comment]
all domains are already pointing to the new server IP (24hrs already).
stream domains are not working in https, only in http.

**Comment by n1ce-0ne** [comment]
one thing to note: https://test.example.com:8000/ would never work.

if your using a port number with a url it must be like: http://test.example.com:8000/

cant you set the new server up exactly like the old where it worked ?

**Comment by mdrxy** [comment]
There's not much anyone here could do to help out without getting more information regarding your installation config, server environment, network setup, etc.

**Comment by bombelman** [comment]
> one thing to note: https://test.example.com:8000/ would never work.
not sure why you claim this.
Please run in VLC:
https://ac.surilive.com:8000 this is a new url, it is working fine.
however all sites/apps use the old url: https://my.surilive.com:8150 
in VLC it will give a https error (don't accept it)
http plays just fine: http://my.surilive.com:8150

the IP for my.surilive.com has already been pointed to the new server.


> if your using a port number with a url it must be like: http://test.example.com:8000/
https also works, always has amigo (see new examplke as test: https://ac.surilive.com:8150)

> cant you set the new server up exactly like the old where it worked ?
it is an exact copy.
I't just on a different server, with a different IP.
How should I have set the server up?

**Comment by bombelman** [comment]
![Screenshot 2024-10-15 185151](https://github.com/user-attachments/assets/57b850a5-ac57-4377-8fdc-087170d6e628)

Already did this....

**Comment by bombelman** [comment]
so here's what I think, after reading some.
the cert needs a forced "renew".
is this possible in or out of AzuraCast?

**Comment by bombelman** [comment]
![image](https://github.com/user-attachments/assets/03b17e69-8d98-4bc6-8031-472b0ab47be3)
https://my.surilive.com:8000 is giving an error.
This url worked fine for the previous server.

![image](https://github.com/user-attachments/assets/fd6670a2-2b5d-4d5a-b6ef-27fd0fe4daf1)
https://ac.surilive.com:8000 is giving me the secure page of Shoutcast status.

**Comment by bombelman** [comment]
bumpin' this one

**Comment by BusterNeece** [comment]
@bombelman Just a long shot here, but did you try taking the spaces out from between the domains, i.e. `domain1,domain2`? It shouldn't need that, but maybe it does.

Also, while I appreciate the urgency **you** have with regard to this situation, please keep in mind that just because a situation is urgent for you does not mean that you should transfer that same urgency to the creators of the software. We do our best, and you absolutely must be patient with us to be able to continue to use our community resources.

**Comment by bombelman** [comment]
hi @BusterNeece 
I understand for sure.
Just hoping it migjt catch someones attention with prior knowledge or expwrience dealing with this.

Regarding the spaces, I'v check this,
and even when removing the spaces, they are added back automatically...

**Comment by BusterNeece** [comment]
@bombelman If you go to System Debugger, under the synchronized tasks, there's a task called "RenewAcmeCert"...if you manually run that, it should force a renewal no matter what. Try that.

**Comment by bombelman** [comment]
@BusterNeece tried it but no luck yet.
Just a reminder, the https for web port(s) work.

It's the part for the streaming ports that fail https (eg https://my.surilive.com:8000)
Are the DNAS streaming ports handled differently?

**Comment by BusterNeece** [comment]
@bombelman Remember not to post things like "no luck" here if you want expedited support. Post the actual message running the task gave you. It should've been a popup with a log of exactly what it did.

If it was successfully able to renew the cert, then you may just need to go through the stations and manually full-restart them to kick over to the new cert.

**Comment by bombelman** [comment]
restarted as suggested, no change (yet) but I will retry again later to see if the cert has updated.
After running the task manually, I got "successful" and "completed" in the log output, no other details.


![Screenshot 2024-10-17 080958](https://github.com/user-attachments/assets/39672795-2b87-4522-9251-0246c3d1b8f7)

**Comment by bombelman** [comment]
when visiting https://my.surilive.com:8150 (old url used by all sites/apps) I get a cert error, with visiting https://ac.surilive.com:8150 (new test url, same ip, same server) I don't get that error.

it seems the "my." subdomain does/did not get the update for the DNAS stream ports.
Does the RenewAcmeCertTask also handle all the stream port certs?

![Screenshot 2024-10-17 at 8 04 34 AM](https://github.com/user-attachments/assets/885f3ebb-a97a-4139-80ba-db7e81dde816)

**Comment by BusterNeece** [comment]
@bombelman I think your browser is caching the SSL cert. When I go to either of the above URLs they now work without issue.

**Comment by bombelman** [comment]
indeed, I checked on another PC and it seems the proper cert is now loading, with issue date of today.

the manual run did indeed seems to have done the trick.
what I also did was change the order the domains are listed, and also changed the site base url, might also have played a role but not sure.

Thanks for your assistance @BusterNeece 

![Screenshot 2024-10-17 at 8 49 14 AM](https://github.com/user-attachments/assets/da5f374a-c7fe-4b5c-985b-364d661cffb8)
